README.md

```diff
@@ -39,7 +39,7 @@ Spike supports the following RISC-V ISA features:
   - Svinval extension, v1.0
   - Sdext extension, v1.0-STABLE
   - Sdtrig extension, v1.0-STABLE
-    - 4 triggers support type={2, 3, 4, 5, 6, 15} (mcontrol, icount, itrigger, etrigger, mcontrol6, disabled)
+    - Support trigger type={2, 3, 4, 5, 6, 15} (mcontrol, icount, itrigger, etrigger, mcontrol6, disabled)
```

Comment:
The grammar doesn't sound right here. How about:

    - Triggers support type={2, 3, 4, 5, 6, 15} (mcontrol, icount, itrigger, etrigger, mcontrol6, disabled)

Reply:
Removed the line in https://github.com/riscv-software-src/riscv-isa-sim/pull/1592/commits/6903ff698d2a6469b3773558d760b31a2947ae9f